src/main/java/com/softserve/academy/spaced/repetition/controller/CourseController.java

```diff
@@ -116,7 +138,7 @@
         return new ResponseEntity<>(coursePublicDTO, HttpStatus.CREATED);
     }
 
-    @Auditable(action = AuditingAction.CREATE_COURSE)
+    @Auditable(action = AuditingAction.EDIT_COURSE_ADMIN)
     @PutMapping(value = "/api/cabinet/courses/{courseId}")
```

Comment:
Inconsistency. This url starts by /api/cabinet. while others start by /api/admin.